src/customers/account-dashboard-address-book.md

```diff
@@ -32,4 +32,16 @@ _[Address Book]({% link configuration/customers/customer-configuration.md %})_
     The new address is listed under Additional Address Entries.
 
 ![]({% link images/images-ee/customer-account-dashboard-address-saved.png %}){: .zoom}
-_[Additional Address Entries]({% link configuration/customers/customer-configuration.md %})_{:.ee-only}
+_[Additional Address Entries]({% link configuration/customers/customer-configuration.md %})_
+
+## Change an address
+
+1. In the sidebar of the customer account, select **Address Book**.
+
+1. Choose one of the methods to edit:
+
+   - Click the **Change Billing/Shipping Address** in the Default Addresses block.
+
+   - Ckick the **Edit** in the **Additional Address Entries** grid.
+
+1. Make changes necessary, and click <span class="btn">Save Address</span>.
```

Comment:
I would also recommend adjusting this line a bit.
```suggestion
1. Make the necessary changes, and click <span class="btn">Save Address</span>.
```

Reply:
Done, please check 😊